covid_model/ode_builder.py

```diff
@@ -211,7 +198,7 @@ def get_default_cmpt_by_attrs(self, attrs):
         return tuple(attrs[attr_name] if attr_name in attrs.keys() else attr_list[0] for attr_name, attr_list in self.attributes.items())
 
     # set a parameter (if val is provided; otherwise apply a multiplier or exponent)
-    def set_param(self, param, val=None, attrs=None, trange=None, mult=None, pow=None, except_attrs=None):
+    def set_param(self, param, val=None, attrs=None, trange=None, mult=None, pow=None, except_attrs=None, desc=None):
```

Comment:
`desc` doesn't do anything, but this method reads in all of the arguments from the jsons in attribute_multipliers.json, and I added "desc" keys as a hacky way to get comments in there.